spdx/writers/jsonyamlxml.py

```diff
@@ -122,7 +123,7 @@ def external_reference_as_dict(external_ref: ExternalPackageRef) -> dict:
             external_ref_dict["comment"] = external_ref.comment
         return external_ref_dict
 
-    def create_package_info(self, package):
+    def create_package_info(self, package, annotation_objects):
```

Comment:
I would probably name this `annotations_by_spdx_id`

Reply:
renamed